components/Header.vue

```diff
@@ -1,8 +1,13 @@
 <template>
   <header class="w-full p-4 shadow-md sticky top-0 bg-surface-primary z-10">
-    <div class="desktop-gutters flex items-center justify-center space-x-2">
-      <icons-centrapay-logo class="w-12 h-12 fill-brand-accent" />
-      <span class="hidden md:block type-headline-3">Centrapay Docs</span>
+    <div class="flex items-center justify-start space-x-2">
+      <NuxtLink
+        to="/"
+        class="flex items-center"
+      >
+        <icons-centrapay-logo class="w-10 h-10 fill-brand-accent" />
```

Comment:
Instead of `w-10 h-10` you could use the class `icon-3xl` which has the same properties.
This means if we ever change our stylesheet everything will be consistent.